config/manager/manager.yaml

```diff
@@ -113,6 +113,8 @@ spec:
             value: "2000"
           - name: NNF_STORAGE_PROFILE_NAMESPACE
             value: nnf-system
+          - name: NNF_CONTAINER_PROFILE_NAMESPACE
```

Comment:
If this is a requirement we should make sure it's in the RFC (description and sample). Maybe even make validating webhook?

@roehrich-hpe How did you handle this requirement?

Reply:
I do have plans to add a webhook in another PR.

Reply:
I have a webhook for nnfstorageprofile, but it doesn't validate that the NNF_STORAGE_PROFILE_NAMESPACE is set.
I suppose it could do that.

Blake, it's possible that your validation steps can be expressed with CEL and that you wouldn't need a webhook.  See things like:
https://kubernetes.io/blog/2022/09/29/enforce-immutability-using-cel/
https://blog.rewanthtammana.com/kubernetes-crd-validation-with-cel-and-kubebuilder-marker-comments

That would be better than a webhook, if CEL can express everything you require.

Reply:
Thanks for those pointers, @roehrich-hpe. I've made a ticket for webhook/CEL.